### PR TITLE
Update lex.dd

### PR DESCRIPTION
**Pull request by jibal**
Minor corrections and language improvements.

### CODE REVIEW

File: lex.dd [hunk 1]
```diff
@@ -746,24 +735,23 @@ $(GNAME HexLetter):
     $(B D)
     $(B E)
     $(B F)
-    $(D _)
 )
 
-	$(P Integers can be specified in decimal, binary, octal, or hexadecimal.)
+	$(P Integers can be specified in decimal, binary or hexadecimal.)
```
Comment (aG0aep6G) [inline lex.dd]: Comma after "binary"?

**Comment by aG0aep6G** [comment]
Aside from that one comma, LGTM.

File: lex.dd (hunk 1 above)
Comment (jibal) [inline lex.dd]: I deleted the comma accidentally.

I'm new to this ... I made this edit from the "Improve this page" button at http://dlang.org/lex.html
Can someone add the comma for me, or do I need to resubmit this, or ... ?

File: lex.dd [hunk 2]
```diff
@@ -892,10 +880,9 @@ $(GNAME LeadingDecimal):
 
 	$(P Floats can be in decimal or hexadecimal format.)
 
-	$(P Hexadecimal floats are preceded with a $(B 0x) and the
-	exponent is a $(B p)
-	or $(B P) followed by a decimal number serving as the exponent
-	of 2.
+	$(P Hexadecimal floats are preceded with a $(B 0x) or $(B 0X) and the
```
Comment (jibal) [inline lex.dd]: Another fix should be "preceded with" -> "preceded by"

File: lex.dd (hunk 1 above)
Comment (aG0aep6G) [inline lex.dd]: I think on the "Files changed" tab you can click the pencil icon (top right corner of the source code section) to make further changes.

**Comment by DmitryOlshansky** [comment]
LGTM

**Comment by DmitryOlshansky** [comment]
Time to wrap up this ;)
@jibal Thanks!

**Pull request merged**